qt/python/mantidqt/widgets/sliceviewer/peaksviewer/presenter.py

```diff
@@ -20,16 +20,19 @@ class PeaksWorkspaceDataPresenter(TableWorkspaceDataPresenter):
     appropriately
     """
     FLOAT_FORMAT_STR = '{:.5f}'
+    DATA_SORT_ROLE = 2001
```

Comment:
Just a small nit, could we comment this as I'm sure it's something Qt specific.

Searching for `Data sort role qt` or `enable_sorting qt` and similar isn't yielding results on my search engine so I'm a bit in the dark what this does

Reply:
Yes that's a good point. I'll add a clarifying comment.